tools/pythonpkg/src/vector_conversion.cpp

```diff
@@ -246,6 +278,24 @@ void VectorConversion::NumpyToDuckDB(PandasColumnBindData &bind_data, py::array
 		}
 		break;
 	}
+	case PandasType::CATEGORY: {
+		D_ASSERT(py::hasattr(numpy_col, "dtype"));
+		auto src_type = string(py::str(numpy_col.attr("dtype")));
+		if (src_type == "int8") {
+			ScanPandasCategory<int8_t>(numpy_col, count, offset, out);
+		} else if (src_type == "int16") {
+			ScanPandasCategory<int16_t>(numpy_col, count, offset, out);
+		} else if (src_type == "int32") {
+			ScanPandasCategory<int32_t>(numpy_col, count, offset, out);
+		} else if (src_type == "int64") {
```

Comment:
This is an unhandled situation, but should it be handled here or in `ScanPandasCategory` ?